src/dso_api/urls.py

```diff
@@ -15,7 +16,8 @@
         views.multiple_slashes,
         name="error-trailing-slashes",
     ),
-]
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
added this because docker uses uwsgi, in which case the Django DEBUG magic doesnt apply